.github/workflows/dockerfiles/alpine

```diff
@@ -1,9 +1,16 @@
 FROM alpine:3.14.0
 
-RUN apk add m4 g++ make pkgconf git tar xz gmp-dev re2-dev sqlite-dev fuse-dev ncurses-dev dash sqlite-static ncurses-static linux-headers
+RUN apk add m4 g++ make pkgconf git tar xz gmp-dev re2-dev sqlite-dev fuse-dev ncurses-dev dash sqlite-static ncurses-static linux-headers curl
```

Comment:
Is curl used?

Reply:
oh good catch, I think I was at one point but then I switch to wget, let me go purge curl